tests/Fixer/FunctionNotation/FopenFlagsFixerTest.php

```diff
@@ -102,57 +102,41 @@ public static function provideFixCases(): iterable
                 ',
             ['b_mode' => false],
         ];
+
+        foreach (self::provideDoNotFixCodeSamples() as $name => $code) {
+            yield $name.' with b_mode' => [$code];
+
+            yield $name.' without b_mode' => [$code, null, ['b_mode' => false]];
+        }
     }
 
     /**
-     * @dataProvider provideDoNotFixCases
+     * @return iterable<string>
```

Comment:
```suggestion
     * @return iterable<string, string>
```
but key must be added to the last case.